packages/webcomponents/src/components/business-form/business-form-stepped.tsx

```diff
@@ -55,16 +43,31 @@ export class BusinessFormStepped {
     this.fetchData = this.fetchData.bind(this);
   }
 
+  componentStepMapping = {
+    0: (formController) => <justifi-business-generic-info formController={formController} />,
+    1: (formController) => <justifi-legal-address-form formController={formController} />,
+    2: (formController) => <justifi-additional-questions formController={formController} />,
+    3: (formController) => <justifi-business-representative formController={formController} />,
+    4: (formController) => <justifi-business-owners formController={formController} />
+  };
+
   componentWillLoad() {
     if (!this.authToken) {
       console.warn(
         'Warning: Missing auth-token. The form will not be functional without it.',
       );
+      this.errorMessage = 'Missing auth-token. The form will not be functional without it.';
+    }
+    if (!this.businessId) {
+      console.warn(
+        'Warning: Missing business-id. The form requires an existing business-id to function.'
+      )
+      this.errorMessage = 'Missing business-id. The form requires an existing business-id to function.';
```

Comment:
Let's not set user-facing errors here. An implementing dev should test their implementation and see the console errors but I don't think this should ever make it to the user